src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.controller.js

```diff
@@ -157,7 +157,13 @@ angular.module("umbraco")
             }
 
             function upload(v) {
-                angular.element(".umb-file-dropzone .file-select").trigger("click");
+                var fileSelect = angular.element(".umb-file-dropzone .file-select");
+                if (fileSelect.length === 0){
+                    localizationService.localize('media_uploadNotAllowed').then(function (message) { notificationsService.warning(message); });
+                }
+                else{
+                    angular.element(".umb-file-dropzone .file-select").trigger("click");
```

Comment:
@olibos you don't need find the element again. It is already available in `fileSelect` variable.

Reply:
My bad! First victim of daylight saving time 😂
Fixed in new version 😉

Reply:
@bjarnef have you seen the new commit?

Reply:
@olibos yes I did... much better 😁 👍